models/staging/stg_mixpanel__event.sql

```diff
@@ -12,6 +12,7 @@ fields as (
     select
         cast( {{ dbt_utils.date_trunc('day', 'time') }} as date) as date_day,
         lower(name) as event_type,
+        cast(time as {{ dbt_utils.type_timestamp() }} )as occurred_at,
```

Comment:
Missing space after the end `)`, the compiled code currently looks like:
```
cast(time as 
    timestamp
 )as occurred_at,
```

Reply:
ah good catch -- fixed!